.github/workflows/aws_tfhe_multi_gpu_tests.yml

```diff
@@ -85,9 +87,10 @@ jobs:
             echo "HOME=/home/ubuntu";
           } >> "${GITHUB_ENV}"
 
-      - name: Run core crypto, integer and internal CUDA backend tests
+      # No need to test core_crypto and classic PBS in integer since it's already tested on single GPU.
+      - name: Run multi-bit CUDA integer tests
         run: |
-          make test_gpu
```

Comment:
we remove some tests ?

Reply:
Yes, seen with @agnesLeroy

Reply:
Yes there is no multi-GPU support in core-crypto and in the cuda backend, only at the integer level. We only need to test the multi-bit integer ops, since the multi-gpu logic is implemented at the integer level if the classical PBS breaks we'll see it in single GPU tests.

Reply:
ah indeed